Bottom: d17f42a5fb83b2364014b640d81c71f6ceb05760
Top:    cf201ed5ab78513fc28b8180be332823d352668d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-12-27 19:30:46 +0000

Refresh of crasy-test

---

diff --git a/src/draw_funcs.c b/src/draw_funcs.c
index 6cb9c01..acc6259 100644
--- a/src/draw_funcs.c
+++ b/src/draw_funcs.c
@@ -300,7 +300,7 @@ draw_layer (LayerType *layer, const BoxType *drawn_area, void *userdata)
     return;
 
   r_search (layer->line_tree, drawn_area, NULL, line_callback, layer);
-  r_search (layer->arc_tree, drawn_area, NULL, arc_callback, layer);
+  r_search (layer->arc_tree, drawn_area,  NULL, arc_callback,  layer);
   r_search (layer->text_tree, drawn_area, NULL, text_callback, layer);
 
   /* We should check for gui->gui here, but it's kinda cool seeing the
